sources/standard_pipeline.py

```diff
@@ -0,0 +1,111 @@
+import os
+from io import BytesIO
+from typing import Any
+
+import dlt
+
+try:
+    from .standard.filesystem import filesystem_resource  # type: ignore
+except ImportError:
+    from standard.filesystem import filesystem_resource
+
+import pandas as pd
+from dlt.extract.source import TDataItem, TDataItems
+
+
+@dlt.transformer(name="filesystem")
+def copy_files(
+    items: TDataItems,
+    storage_path: str,
+) -> TDataItem:
+    """Reads files and copy them to local directory.
+
+    Args:
+        items (TDataItems): The list of files to copy.
+        storage_path (str, optional): The path to store the files.
+
+    Returns:
+        TDataItem: The list of files copied.
+    """
+    storage_path = os.path.abspath(storage_path)
+    os.makedirs(storage_path, exist_ok=True)
+    for file_obj in items:
+        file_dst = os.path.join(storage_path, file_obj["file_name"])
+        file_obj["path"] = file_dst
+        with open(file_dst, "wb") as f:
+            f.write(file_obj.read())
+        yield file_obj
+
+
+@dlt.transformer(
+    name="filesystem",
+    write_disposition="merge",
+    merge_key=None,
+    primary_key=None,
+)
+def extract_parquet(
+    items: TDataItems,
+) -> TDataItem:
+    """Reads files and copy them to local directory.
+
+    Args:
+        items (TDataItems): The list of files to copy.
+
+    Returns:
+        TDataItem: The file content
+    """
+    for file_obj in items:
+        df = pd.read_parquet(file_obj.open_fs())
```

Comment:
heh what's this? OK it makes sense to show how to read parquet but I think you want to use pyarrow?